pkg/config/config.go

```diff
@@ -55,6 +55,17 @@ func (c *Config) GetBaseDir() string {
 	return filepath.Dir(c.configPath)
 }
 
+// GetChart returns a dependency chart configuration.
+func (c *Config) GetChart(logger *slog.Logger, chart string) (*Dependency, error) {
```

Comment:
Given this method returns a `&Dependency{}`, let's name it `GetDependency` instead?

Reply:
With your latest changes, I think I can rollback that change.

Reply:
It is in fact still needed so I've renamed it.